paper.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'A Python Project Template for Healthy Scientific Software'
+tags:
+  - Python
+  - template
+authors:
+  - name: Drew Oldag
+    orcid: asdfasdf
+    equal-contrib: true
+    affiliation: "1, 2"
+  - name: Melissa DeLucchi
+    orcid: asdfasdf
+    equal-contrib: true
+    affiliation: "1, 3"
+  - name: Wilson Beebe
+    orcid: asdfasdf
+    affiliation: "1, 2"
+  - name: Doug Branton
+    orcid: asdfasdf
+    affiliation: "1, 2"
+  - name: Sandro Campos
+    orcid: asdfasdf
+    affiliation: "1, 3"
+  - name: Carl Christofferson
+    orcid: asdfasdf
+    affiliation: "1, 2"
+  - name: Andrew Connolly <-----------????
+    orcid: asdfasdf
+    affiliation: "1, 2"
+  - name: Jeremy Kubica
+    orcid: asdfasdf
+    affiliation: "1, 3"
+  - name: Olivia Lynn
+    orcid: asdfasdf
+    affiliation: "1, 3"
+  - name: Kostya Malanchev
+    orcid: asdfasdf
+    affiliation: "1, 3"
+  - name: Alex Malz <----------????
```

Comment:
Fairly confident that Alex didn't contribute to the PPT, but please correct me if I'm forgetting or not considering something.